api/v1alpha1/usersignup_types.go

```diff
@@ -196,6 +196,51 @@ type UserSignupSpec struct {
 	// a new IdP provider client, and contains the user's "original-sub" claim
 	// +optional
 	OriginalSub string `json:"originalSub,omitempty"`
+
+	// IdentityClaims contains as-is claim values extracted from the user's access token
+	// +optional
+	IdentityClaims IdentityClaimsEmbedded `json:"identityClaims,omitempty"`
+}
+
+// IdentityClaimsEmbedded is used to define a set of SSO claim values that we are interested in storing
+// +k8s:openapi-gen=true
+type IdentityClaimsEmbedded struct {
+
+	// PropagatedClaims
+	PropagatedClaims PropagatedClaims `json:",inline"`
```

Comment:
I thought that we wanted to embed the PropagatedClaims to make any future changes in the set of the fields easier.
```suggestion
	PropagatedClaims `json:",inline"`
```

Reply:
Yes! Good catch.

Reply:
That is a bit nicer, good suggestion.